lib/wab.rb

```diff
@@ -1,6 +1,12 @@
 
 # Web Application Builder
 module WAB
+
+  # Returns a file contents from the gem export directory.
+  def self.get_export(path)
+    File.open(File.expand_path("#{__dir__}/../export#{path}")) { |f| f.read() }
```

Comment:
This looks fishy.. 
`../` in the middle of a path-string.. ?!

Consider using `File.join` for proper path-resolution..?

Reply:
I think we decided that File.join always uses / so the string should be fine. A check for inside export is need though.

Reply:
yep sanitization is better..